tests/run.sh

```diff
@@ -33,8 +35,16 @@ if [[ ! -f broken_update.ext4 ]]; then
     dd if=/dev/urandom of=broken_update.ext4 bs=10M count=5
 fi
 
+if [[ -z $AWS_ACCESS_KEY_ID ]] || [[ -z $AWS_SECRET_ACCESS_KEY ]]; then
+    echo "AWS credentials (AWS_SECRET_ACCESS_KEY / AWS_SECRET_ACCESS_KEY) are not set, not running S3 tests"
+else
+    sed -i "s/%AWS_KEY%/${AWS_ACCESS_KEY_ID}/" ../docker-compose.s3.yml
```

Comment:
so if you replace %..% with ${AWS_KEY} etc you could just export AWS_KEY AWS_... here